tracker/index.js

```diff
@@ -1,8 +1,15 @@
 import 'promise-polyfill/src/polyfill';
 import 'unfetch/polyfill';
-import { post, hook, doNotTrack } from '../lib/web';
+import { doNotTrack, hook, post } from '../lib/web';
 import { removeTrailingSlash } from '../lib/url';
 
+function log(isDebug, messageFn) {
+  if (isDebug) {
+    return;
+  }
+  console.log(messageFn());
+}
+
```

Comment:
I'd rather not have debugging in the script. It needs to be as small as possible.

Reply:
ok, makes sense :)